place_tracker/lib/place_map.dart

```diff
@@ -221,14 +233,6 @@ class _PlaceMapState extends State<PlaceMap> {
       // Add the new place to the places stored in appState.
       final newPlaces = List<Place>.from(appState.places)..add(newPlace);
 
-      // Manually update our map configuration here since our map is already
-      // updated with the new marker. Otherwise, the map would be reconfigured
-      // in the main build method due to a modified AppState.
-      _configuration = MapConfiguration(
-        places: newPlaces,
-        selectedCategory: appState.selectedCategory,
-      );
```

Comment:
Removed because this reconfiguration already happens inside `_maybeUpdateMapConfiguration`